crates/optimism/rpc/src/eth/call.rs

```diff
@@ -149,7 +149,12 @@ where
                 .map(|v| v.saturating_to())
                 .unwrap_or_default(),
             // EIP-7702 fields
-            authorization_list: authorization_list.unwrap_or_default(),
+            authorization_list: authorization_list
+                .unwrap_or_default()
+                .into_iter()
+                .map(Either::Left)
+                .collect(),
```

Comment:
hmm, this is a bit horrible, 

I don't think it's useful to do this for auths individually, but this is a revm api discussion